<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sideMenu</class>
 <widget class="QWidget" name="sideMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>sideMenu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>40</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="ButtonIcon" name="import_profile_button">
      <property name="text">
       <string>Import profile</string>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ButtonIcon" name="proxies_button">
      <property name="text">
       <string>Proxies</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ButtonIcon" name="certif_button">
      <property name="text">
       <string>Certificates &amp;&amp; Tokens</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ButtonIcon" name="settings_button">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ButtonIcon" name="statistic_button">
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>230</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="version_label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Version 1.0.0 ( XXX )</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonIcon</class>
   <extends>QPushButton</extends>
   <header>button_icon.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
